### PR TITLE
Updated code for unsupported songs and song with movie tag

### PR DESCRIPTION
**Pull request by vishal2612200**
**[FIX]**

**In raising this pull request, I confirm the following (please check boxes):**

- [x] I have read and understood the [contributors guide](https://github.com/SwagLyrics/SwagLyrics-For-Spotify/blob/master/.github/CONTRIBUTING.md).
- [x] I have checked that another pull request for this purpose does not exist.
- [x] I have considered, and confirmed that this submission will be valuable to others.
- [x] I accept that this submission may not be used, and the pull request closed at the will of the maintainer.
- [x] I give this submission freely, and claim no ownership to its content.

**My familiarity with the project is as follows (check one):**

- [ ] I have never used the project.
- [x] I have used the project briefly.
- [ ] I have used the project extensively, but have not contributed previously.
- [ ] I am an active contributor to the project.

---

This is a fix to most of the issues related to Bollywood movie songs e.g ghungoo, zikr and so on. I has verified this fix on my local computer and some of the result screenshot has been pasted on a different issue section as a reply. I hope by updating this code, we can close a lot of issues on swaglyrics. 
 
issue reference:
#2090, #2198, #2093, #2094, #2053, #2055 and some more similar issue

### CODE REVIEW

**Comment by flabbet** [comment]
Make sure that tests are passing

File: swaglyrics/cli.py [hunk 1]
```diff
@@ -69,6 +69,14 @@ def get_lyrics(song, artist):
 	url_data = stripper(song, artist)  # generate url path using stripper()
 	if url_data.startswith('-') or url_data.endswith('-'):
 		return None  # url path had either song in non-latin, artist in non-latin, or both
+	if 'From' in url_data:
```
Comment (aadibajpai) [inline swaglyrics/cli.py]: This is an unnecessarily complex solution to this issue

File: swaglyrics/cli.py [hunk 2]
```diff
@@ -96,12 +96,14 @@ def lyrics(song: str, artist: str, make_issue: bool = True) -> str:
 	:param make_issue: whether to make an issue on GitHub if song unsupported
 	:return: lyrics if song playing
 	"""
+	"""
```
Comment (aadibajpai) [inline swaglyrics/cli.py]: why is this necessary?

File: swaglyrics/cli.py [hunk 3]
```diff
@@ -96,12 +96,14 @@ def lyrics(song: str, artist: str, make_issue: bool = True) -> str:
...
 	try:
 		with open(unsupported_txt, encoding='utf-8') as unsupported:
 			if f'{song} by {artist}' in unsupported.read():
 				return f'Lyrics unavailable for {song} by {artist}.\n'
 	except FileNotFoundError:
 		pass
+	"""
```
Comment (aadibajpai) [inline swaglyrics/cli.py]: same as above

File: swaglyrics/cli.py [hunk 4]
```diff
@@ -13,7 +13,7 @@ def clear() -> None:
 	os.system('cls' if os.name == 'nt' else 'clear')  # clear command window
 
 
-brc = re.compile(r'([(\[](feat|ft)[^)\]]*[)\]]|- .*)', re.I)  # matches braces with feat included or text after -
+brc = re.compile(r'([(\[](feat|ft|From)[^)\]]*[)\]]|- .*)', re.I)  # matches braces with feat included or text after -
```
Comment (aadibajpai) [inline swaglyrics/cli.py]: This technically works but it is a bit too generalized, we know the from in the song is something like (From "<movie name>") so we should check for the quotes too, to avoid possible false positives.

File: swaglyrics/cli.py (hunk 2 above)
Comment (vishal2612200) [reply swaglyrics/cli.py]: Because After commenting out the code, Code does required functioning for the unsupported song also, after the addition of 'From' inside the regex expression.

File: swaglyrics/cli.py (hunk 4 above)
Comment (vishal2612200) [reply swaglyrics/cli.py]: ya, you are right, but when we try to check quotes too. It became a little bit of a complex solution.

Comment (aadibajpai) [reply swaglyrics/cli.py]: yeah that's okay
